### ISSUE TITLE
"Coin not found" error on ETH2x-FLI

### ISSUE THREAD

**Issue by 0xEval** [Eloise1988/COINGECKO#9]
Good afternoon Eloise,

The following Gsheet function does not seem to fetch the price `=GECKOPRICE(B4,"USD")` where `B4 = "eth2x-fli"`. It used to work fine prior to the May-2021 release that I have just updated to.

There is perhaps something wrong with the coin caching logic that you have added.

NB: the asset is listed in http://api.charmantadvisory.com/COINGECKOID/json

Not quite sure what is wrong.

**Comment by 0xEval** [comment]
`curl -X GET "https://api.coingecko.com/api/v3/coins/eth-2x-flexible-leverage-index" -H "accept: application/json"` 

works fine. I've tried both ids/tickers `eth2x-fli` and `eth-2x-flexible-leverage-index` but none work

**Comment by Eloise1988** [comment]
<img width="581" alt="Screen Shot 2021-05-06 at 15 34 05" src="https://user-images.githubusercontent.com/53000607/117307139-98f25f80-ae80-11eb-8515-208551cb037b.png">

I updated the code. Coins that had "-" in their ticker were mistaken as a currency. So I changed it like shown on the image.

Also Coins with "-" have long ids so if you request a list of tickers, make sure to limit it to 5 tickers otherwise it will return an error.

Here is the new code for CoinGeckov2:
https://raw.githubusercontent.com/Eloise1988/COINGECKO/master/CoinGeckoV2.gs
